android/build.gradle

```diff
@@ -39,6 +39,7 @@ android {
         versionName "2.0.1"
 
         consumerProguardFiles 'consumer-rules.pro'
+        missingDimensionStrategy 'dependencyVersions', 'latestDependencies', 'unityIAP'
```

Comment:
Could we add a comment here specifying what it does? 

My understanding is that this will try to choose the latest dependencies first, but if there's an incompatibility with other requirements for the same app, it'll try the flavor `unityIAP`, is that correct?